views/result.hbs

```diff
@@ -0,0 +1,33 @@
+<section class="mt5">
+  <ul class="pa2 ma0">
+    {{#each results}}
+    <li class="list tc pt6 ma1 br2">
+      <div class="bg-black-50 white ba br2 b--transparent ma0 pa0">
+         <div class="dib tl w-40">
+           <h1 class="ma1 f2">{{this.name}}</h1>
+         </div>
+         <div class="dib tl w-40">
+           <h1 class="ma1 f4">{{this.loc}}</h1>
+         </div>
+         <div class="dib tl w-40">
+           <h1 class="ma1 f4">{{this.category}}</h1>
```

Comment:
Do we want to show the category in the results page? in the protoype we had to only show:

- the name of the business
- it's address
- the accessibility options

I think having the opening hours is good too, but don't think we need the category